CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # New Relic Ruby Agent Release Notes
 
+
+## dev
+
+Version <dev> allows the agent to record the http status code on a transaction when middleware instrumentation is disabled and fixes a bug in `NewRelic::Rack::AgentHooks.needed?`.
```

Comment:
Since we're now including content type...
```suggestion
Version <dev> allows the agent to record additional response information on a transaction when middleware instrumentation is disabled and fixes a bug in `NewRelic::Rack::AgentHooks.needed?`.
```

Reply:
Oh good catch I hadn't thought about the summary, thank you!